pkgs/games/xonotic/default.nix

```diff
@@ -88,6 +90,7 @@ let
         install -Dm644 ../../misc/logos/xonotic_icon.svg \
           $out/share/icons/hicolor/$size/xonotic.svg
       done
+      (cd ../d0_blind_id && make install)
```

Comment:
So we build d0_blind_id for every variant but only install it on the GLX variant? I don't understand.

Reply:
That is in the general section before the lib.optionalString stanzas.

Reply:
Ohhh GitHub collapsed the diff at that point. Nevermind this comment.